user_docs/en/userGuide.t2t

```diff
@@ -267,6 +267,70 @@ To open NVDA's menu, press ``NVDA+n``.
 To open NVDA's general settings dialog directly, press ``NVDA+control+g``.
 Many settings screens have keystrokes to open them directly, such as ``NVDA+control+s`` for synthesizer, or ``NVDA+control+v`` for other voice options.
 
+++ Add-ons ++[Addons]
+Add-ons are programs which provide new or changed functionality for NVDA.
+Add-ons are developed by the NVDA community, or external companies and are unaffiliated with NV Access.
+
+As with any software, it is important to be confident in the developer of an add-on before using it.
+The first time the add-on store is opened, NVDA displays a warning about add-ons.
+Add-ons are not vetted by NV Access and may have unrestricted functionality and access to information.
+Press ``spacebar`` if you have read the warning and do not need to see it next time.
+Press ``enter`` to accept the warning and proceed to the Add-on Store.
+The "[Add-ons and the Add-on Store #AddonsManager]" section of the user guide contains information about every feature of the add-on store.
+
+
+The Add-on Store is available from the Tools menu.
+Press ``NVDA+n`` to open the NVDA menu, then ``t`` for tools, then ``a`` for add-on store.
+When the add-on store opens, it shows "Available add-ons" if no add-ons are installed.
+When add-ons are installed, the Add-on Store opens to the "Installed add-ons" tab.
+
++++ Available add-ons +++[AvailableAddons]
+When the window first opens, add-ons may take a few seconds to load.
+NVDA will read the name of the first add-on once the list of add-ons finishes loading.
+Available add-ons are listed alphabetically in a grid.
+Use the ``arrow keys`` or press the first letter of an add-on name to move around the list.
+Press ``tab`` once to move to a description of the currently selected add-on.
+Use the reading or ``arrow keys`` to read the full description.

```

Comment:
We don't give instructions like this through most of the user guide (that's more a training module area).  I feel saying something like this is needed here until #16485 is implemented (if that were implemented, I think we could drop this line).  In the meantime, I've suggested"reading text" which links to the quick start guide topic of that name - https://www.nvaccess.org/files/nvda/documentation/userGuide.html#ReadingText  Otherwise we could say "Use the Say all, or navigation keys"?
```suggestion
Use the [reading text #ReadingText] or ``arrow keys`` to read the full description.

```

Reply:
@Qchristensen I'm not sure "use the reading text" makes sense. It's like "Use the force", without there being a force. :-)

I get that you mean it to be read "Use the reading text keys, or the arrow keys", but even that sounds awkward since "reading text keys" aren't a thing.
"reading keys", okay, as a general shorthand for "review or object nav keys", but "reading text keys"? Hmmm.

Reply:
I would like an excuse to write "Use the force" in the user guide somewhere!  Happy to accept a change here - I was originally thinking of saying use "Say all", but that is just one of any number of keys you COULD use to read text with.  I don't think there are (many, or any?) other examples of this kind of instruction in the User Guide, as it's not how it is written.  If we could have NVDA read that whole text field on focus it would negate the need to give any specific instruction here :)  I wasn't keen to leave it unsaid though as it is easy to get to that description field, have it read one line and not know whether there is more or that you need to specifically read it if there is.

I just had a look in Basic Training and I mention "Reading keys" or "Reading and navigation keys" in a couple of places.

Reply:
Perhaps this is more of a discussion for #16485 but for me NVDA reports that this field is multiline, which pretty clearly indicates  there may be more than one line there. Why  it is a problem in the add-on store specifically and not in many other places - when tabbing into an multiline edit field on web pages for example?